src/lib/modules/Patron/PatronBulkExtendLoans/actions.js

```diff
@@ -0,0 +1,56 @@
+import { loanApi } from '@api/loans';
+import { searchReady } from '@api/utils';
+import {
+  sendErrorNotification,
+  sendSuccessNotification,
+} from '@components/Notifications';
+import {
+  fetchPatronCurrentLoans,
+  IS_LOADING as currentLoansIsLoading,
+} from '../PatronCurrentLoans/actions';
+import _isEmpty from 'lodash/isEmpty';
+
+export const IS_LOADING = 'bulkLoanExtend/IS_LOADING';
+export const SUCCESS = 'bulkLoanExtend/SUCCESS';
+export const HAS_ERROR = 'bulkLoanExtend/HAS_ERROR';
+
+export const bulkLoanExtension = (patronPid) => {
+  return async (dispatch) => {
+    dispatch({
+      type: IS_LOADING,
+    });
+    try {
+      const response = await loanApi.bulkExtendLoans(patronPid);
```

Comment:
Question: can be return from the backend which ones have been extended and which not and present this result to the user?